README.md

```diff
@@ -27,14 +27,19 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 Be prepared to demonstrate your understanding of this week's concepts by answering questions on the following topics. You might prepare by writing down your own answers before hand.
 
 1. What is the DOM?
+Is not Java script, the DOM is the language that we use to interact with the element in the html structure or can also be used to interact in other spaces.
```

Comment:
The DOM is an object representation of your HTML. We can interact with (read, write) the DOM using Javascript.